src/Model/AuthorshipDetector.java

```diff
@@ -0,0 +1,13 @@
+package Model;
```

Comment:
you can create a separate package called Services to separate the business logic classes from the entity classes later on